pages/design/ui-patterns/blankslate.mdx

```diff
@@ -0,0 +1,45 @@
+import {Flex, Box, Text, BorderBox} from '@primer/components'
+
+export const meta = {displayName: 'Blankslate'}
+
+# Blankslate
+Blankslates are used to convey empty states across our products. These guidelines demonstrate best practices for using the Blankslate component. If you're looking for guidelines on implementation, please refer to [Primer CSS](https://primer.style/css/components/blankslate) or [Primer Components](https://primer.style/components).
+
+## The Blankslate Component
+The blankslate is made up of several elements that work together to inform the user about a feature and how to proceed forward. Below are the different elements of the component and how to modify them.
+
+![anatomy](https://user-images.githubusercontent.com/6846673/62644438-f3066f80-b8fe-11e9-8e4b-e8af1cac0e58.png)
+
+### Graphic
+The graphic can bring delight, preview an interface element, or simply add color to balance the information of the component. There are a few variations to the graphic, which are outlined later on this page.
+
+### Primary Text
+This text will be the largest in the component and can make a user feel much more comfortable to engage. Primary text is meant to be welcoming and invitational. Text should sound human and convey the intention of the feature.
+
+### Secondary Text
+This optional text will be located below the primary text and should support informing the user about the feature. Secondary text should be brief and nonredundant if possible. Users should understand the general purpose of the feature and how it might help them accomplish a goal.
```

Comment:
```suggestion
This optional text is used to inform the user about the feature in more detail. Secondary text should be brief and non-redundant if possible. From the text, users should  be able to understand the general purpose of the feature and how it may help them accomplish a goal.
```